x-pack/plugins/cases/common/constants/files.ts

```diff
@@ -5,12 +5,6 @@
  * 2.0.
  */
 
-import type { HttpApiTagOperation, Owner } from './types';
-
 export const MAX_FILE_SIZE = 100 * 1024 * 1024; // 100 MiB
-
-export const constructFilesHttpOperationTag = (owner: Owner, operation: HttpApiTagOperation) => {
-  return `${owner}FilesCases${operation}`;
-};
-
-export const constructFileKindIdByOwner = (owner: Owner) => `${owner}FilesCases`;
+export const MAX_FILES_PER_CASE = 100;
+export const MAX_DELETE_FILES = MAX_FILES_PER_CASE * 5;
```

Comment:
Given that the file deletion is slow what do you think about reducing the number to 50 (the maximum number of files shown on one page in the UI table)?

Reply:
Sounds reasonable to me!